install.sh

```diff
@@ -2,34 +2,34 @@
 
 # Source setting.sh
 source /vagrant/settings.sh
+
+# Fetching LAN_IP and network address
+if [[ ${STANDALONE,,} != yes ]]; then
+   LAN_IP=$(ip addr | grep -Po -m3 'inet \K[\d.]+' | tail -n1)
```

Comment:
network interfaces at my end
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 08:00:27:af:06:9a brd ff:ff:ff:ff:ff:ff
    inet 10.0.2.15/24 brd 10.0.2.255 scope global dynamic enp0s3
       valid_lft 85960sec preferred_lft 85960sec
    inet6 fe80::bc9e:d02b:c104:fcef/64 scope link 
       valid_lft forever preferred_lft forever
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default 
    link/ether 02:42:68:9b:93:36 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever

```
above command gives me `dockers` ip address, which is not what we need.

Check if these snippet helps you. This will not require interface name, no grep, no assumption.

```
akash@skyfall:$ ip route get 8.8.8.8
8.8.8.8 via 10.0.2.2 dev enp0s3  src 10.0.2.15 
    cache 
akash@skyfall:$ ip route get 8.8.8.8|head -n1|cut -d ' ' -f8
```

Reply:
> above command gives me dockers ip address, which is not what we need.

Akash, Deepanshu was assuming that the command was executed inside the VB machine, not on the host. It seems that you are trying the command on the host.

> `ip route get 8.8.8.8`

This does not work for the case of STANDALONE, because it will give the WAN_IP, not the LAN_IP. (By the way, `ip route | grep default` is better than this, when we need to get the WAN_IP).

Reply:
This looks like `ip addr` output of host machine.(because of `docker0`)
In vagrant box also first is `lo`, second is `enp0s3`, third is `enp0s8`(bridged adapter the one we need)

Reply:
>is also good option to avoid hardcoding interface name here.

Yes I was also trying to do the same. See my thoughts were. when we do a `vagrant up`. Output is something like this - 
```
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: bridged
```
So I thought network adapter will always be created in this ordered mentioned.

Reply:
>Yes, interface is the same every time, and it is LAN_IF.
The code proposed by Akash is correct

So should we merge this?